src/nanomoni/cpu_affinity.py

```
@@ -0,0 +1,61 @@
+between cores mid-run and pay an L3 miss on arrival. Owning a core for the
+process's whole life removes that variance, and it can only be expressed from
+inside the container, per process.
+"""
+
+from __future__ import annotations
+
+import fcntl
+import os
+from typing import IO
+
+LOCK_DIR = "/tmp/nanomoni-cpu-pins"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/nanomoni-cpu-pins"

(S108)